addons/point_of_sale/tests/test_frontend.py

```diff
@@ -1187,6 +1187,20 @@ def test_res_partner_scan_barcode(self):
         self.main_pos_config.with_user(self.pos_user).open_ui()
         self.start_tour("/pos/ui?config_id=%d" % self.main_pos_config.id, 'BarcodeScanPartnerTour', login="pos_user")
 
+    def test_refund_few_quantities(self):
+        """ Test to check that refund works with quantities of less than 0.5 """
+        self.env['product.product'].create({
+            'name': 'Sugar',
+            'list_price': 3,
+            'taxes_id': False,
+            'available_in_pos': True,
+            'uom_id': self.env.ref('uom.product_uom_kgm').id,
+            'uom_po_id': self.env.ref('uom.product_uom_kgm').id
+        })
+
+        self.main_pos_config.with_user(self.pos_user).open_ui()
+        self.start_tour("/pos/ui?config_id=%d" % self.main_pos_config.id, 'RefundFewQuantities', login="pos_user", step_delay=100)
```

Comment:
Remove `step_delay`

Reply:
Without it, the step `ProductScreen.pressNumpad("0", ".", "0", "2"),` fails because the numpad buffer is limited to 2 values so it need some time to empty itself.
However it may be possible to reduce the delay or maybe there is a better way to do it?

Reply:
Maybe doing it in 2 steps can be better, and if not maybe add a step between